staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -108,17 +89,20 @@ var expectedPolicy = &audit.Policy{
 	}},
 }
 
-func TestParserV1alpha1(t *testing.T) {
```

Comment:
where did this go? Same for the beta variant.

Reply:
I merged three versions into a loop.